.github/workflows/custom_helpline.yml

```diff
@@ -146,3 +148,4 @@ jobs:
           aws-region: $AWS_REGION
           s3-bucket: $S3_BUCKET
           helpline-name: ${{inputs.helpline_code}}_${{inputs.environment_code}}
+          disable-operating-hours: ${{ (inputs.force_enable_operating_hours || inputs.environment_code == 'PROD') && 'false' || 'true' }}
```

Comment:
This line got me thinking a while. Maybe a comment that explains a bit is useful here?